### DISCUSSION TITLE
Usage without workspace

### DISCUSSION THREAD

**Discussion by rawnly** [vercel/turborepo#2608] [Help] [answered]
As mentioned in the docs:
> Turborepo can be used in any project to speed up the execution of scripts in your package.json.

It works fine for me until I add `dependsOn` keyword on pipelines, then it tries to open `pnpm-workspace.yaml` and fails. Is this behaviour expected?

**Comment by nathanhammond** [comment, marked as answer]
Are you using the topological operator? `^` If so, that is not a valid definition in single package mode.

**Reply by rawnly** [reply]
Hi, no I read the docs, here is my config 

```json
{
    "globalDependencies": [".env"],
    "pipeline": {
        "build": {
            "outputs": ["dist/**", "build/**"],
            "dependsOn": ["lint"],
            "inputs": ["src/**/*.ts"],
            "env": ["NODE_ENV"]
        },
        "dev": {
            "cache": false
        },
        "lint": {
            "inputs": ["src/**/*.ts"],
            "outputs": []
        },
        "coverage": {
            "inputs": ["src/**/*.ts"],
            "outputs": ["coverage/**"],
            "dependsOn": ["build"]
        },
        "test": {
            "outputs": [],
            "inputs": ["src/**/*.ts", "__tests__/**/*.ts"],
            "dependsOn": ["build"]
        }
    }
}

```

**Reply by rawnly** [reply]
[this is the repo](https://github.com/rawnly/ts-package)

**Reply by nathanhammond** [reply]
I'll take a look tomorrow. This may be a bug report!

**Reply by rawnly** [reply]
> I'll take a look tomorrow. This may be a bug report!

any update?
